components/ProfileBody.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Dimensions, StyleSheet, Text, View } from 'react-native';
+import { ButtonGroup } from 'react-native-elements';
+
+var width = Dimensions.get('window').width;
+
+const ProfileBodyStyles = StyleSheet.create({
+  text: {
+    alignSelf: 'center',
+    marginTop: 15,
+  },
+});
+
+const Feed = (props) => {
+  if (props.selectedIndex == 0)
+    return <Text style={ProfileBodyStyles.text}>Communities!</Text>;
+  else
+    return <Text style={ProfileBodyStyles.text}>Likes!</Text>;
+};
+
+export default class ProfileBody extends React.Component {
```

Comment:
It looks like you are using a class component here which is fine, but we will definitely need to decide if we want to standardize function vs class components as we move forwards.